lib/AI-Agent.xml

```diff
@@ -7,22 +7,5 @@
             <JDBC enabled="true"/>
             <MaxStatementQueryLimitInMS>1000</MaxStatementQueryLimitInMS>
         </BuiltIn>
-        <Class name="uk.gov.hmcts.ccd.data.jurisdiction.JurisdictionRepository" type="Data">
-            <Method name="findEntityById"/>
-            <Method name="create"/>
-        </Class>
-        <Class name="uk.gov.hmcts.ccd.data.userprofile.UserProfileAuditEntityRepository" type="Data">
-            <Method name="createUserProfileAuditEntity"/>
-        </Class>
-        <Class name="uk.gov.hmcts.ccd.data.userprofile.UserProfileRepository" type="Data">
-            <Method name="createUserProfile"/>
-            <Method name="updateUserProfile"/>
-            <Method name="findById"/>
-            <Method name="findEntityById"/>
-            <Method name="findAll"/>
-            <Method name="updateUserProfileOnCreate"/>
-            <Method name="deleteJurisdictionFromUserProfile"/>
-            <Method name="findJurisdictionEntityById"/>
-        </Class>
```

Comment:
@timja is it ok to remove these now? how come?

Reply:
This is now using the auto instrumentation feature i demoed to you on Tuesday.
Assuming these are all postgres calls then they aren't needed anymore